app/server/server.go

```diff
@@ -33,9 +33,14 @@ import (
 // InitAndRun initializes the server and runs it
 func InitAndRun(conf config.Reader, dataStore store.DataStore) error {
 	ctx := context.Background()
-	dataStore.LoadWorkflows(ctx)
 
+	if conf.GetBool(dconfig.SettingDebugLog) {
+		log.Setup(true)
+	}
 	l := log.FromContext(ctx)
+
+	dataStore.LoadWorkflows(ctx, l)
```

Comment:
I'm not 100% sure, but is it preferable to pass the logger to the `LoadWorfklows` function instead of passing the context only, and let `LoadWorkflows` obtain its log from the context?

Reply:
if you insist no problem I can change it -- seemed better to pass the logger.

Reply:
Not a big deal